commit b4b9d129bcb44f11a7d36f3a43a48d6f46b3e1fa (refs/changes/56/1056/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-12-05T22:34:05+00:00 (2 years, 10 months ago)
    
    joyent/sdc-docker#102 need to be using bootstrap mode with cueball / moray-client v2
    Reviewed by: Trent Mick <trent.mick@joyent.com>

diff --git a/lib/docker.js b/lib/docker.js
index 8629fe3..d1d05f7 100644
--- a/lib/docker.js
+++ b/lib/docker.js
@@ -367,7 +367,10 @@ App.prototype.createMorayClient = function createMorayClient() {
         host: self.config.moray.host,
         noCache: true,
         port: self.config.moray.port,
-        reconnect: true
+        reconnect: true,
+        dns: {
+            resolvers: [self.config.binder.domain]
+        }
     };
 
     self.log.debug(conf, 'Creating moray client');
diff --git a/sapi_manifests/docker/template b/sapi_manifests/docker/template
index a056637..1bfa4b0 100644
--- a/sapi_manifests/docker/template
+++ b/sapi_manifests/docker/template
@@ -31,6 +31,9 @@
         "host": "{{{MORAY_SERVICE}}}",
         "port": 2020
     },
+    "binder": {
+        "domain": "{{{BINDER_SERVICE}}}"
+    },
     "fwapi": {
         "url": "http://fwapi.{{{datacenter_name}}}.{{{dns_domain}}}"
     },
